package.json

```diff
@@ -45,6 +45,7 @@
     "dice-coefficient": "^1.0.2",
     "eventemitter3": "^1.2.0",
     "express": "^4.13.4",
+    "file-saver": "^1.3.8",
```

Comment:
How big are the BioPAX XML files typically?

Reply:
It is 25KB for a Factoid document that includes 11 different interaction types and 2 participants per interaction. It became 49KB when I made it 22 interactions by adding new elements having the same of existing interaction and participant types with different names.